test/functional/search_controller_test.rb

```diff
@@ -317,6 +319,18 @@ def no_results
     assert_equal "1",       @response.headers["X-Slimmer-Result-Count"]
   end
 
+  test "should pass the search variant through when one is present" do
+    with_variant SearchMatchLength: "B" do
+      get :index, q: "Variant B"
+    end
```

Comment:
This is not clear when the test is or that there even is one. 

I assume a method named `with_variant` would not be responsible for setting a test condition.